apps/bmd/src/utils/tallies.ts

```diff
@@ -6,10 +8,114 @@ import {
   VotesDict,
   YesNoVote,
 } from '@votingworks/types'
-import { CandidateVoteTally, Tally, YesNoVoteTally } from '../config/types'
+import {
+  CandidateVoteTally,
+  MsEitherNeitherTally,
+  Tally,
+  YesNoVoteTally,
+} from '../config/types'
 import { computeTallyForEitherNeitherContests } from './eitherNeither'
 import { getSingleYesNoVote } from './votes'
 
+const combineCandidateTallies = (
+  tally1: CandidateVoteTally,
+  tally2: CandidateVoteTally
+): CandidateVoteTally => {
+  const candidates = []
+  assert(tally1.candidates.length === tally2.candidates.length)
```

Comment:
This would give you more information on a mismatch:
```suggestion
  assert.equal(tally1.candidates.length, tally2.candidates.length)
```